pkg/crc/config/config.go

```diff
@@ -207,6 +212,13 @@ func (c *Config) Get(key string) SettingValue {
 				Invalid: true,
 			}
 		}
+	case uint:
+		value, err = cast.ToUintE(value)
+		if err != nil {
+			return SettingValue{
+				Invalid: true,
+			}
+		}
```

Comment:
Unrelated to this PR, but wondering if we could use go generics to have a generic 'cast' function instead of switching over the types.

Reply:
Right, that would be a good exercise, we can put a tracker issue for it.